app/models/mixins/relationship_mixin.rb

```diff
@@ -708,11 +697,7 @@ def self.deprecate_of_type_and_rel_type_parameter(*args)
   def self.deprecate_start_parameter(*args)
     return args if args.empty? || args.first.kind_of?(Hash)
 
-    unless Rails.env.production?
-      msg = "[DEPRECATION] start parameter is deprecated.  At #{caller[1]}"
-      $log.warn msg
-      warn msg
-    end
+    warn_of_deprecation("start parameter", "")
```

Comment:
Addressed in https://github.com/ManageIQ/manageiq/pull/4731/files#r41756604  Second arg is now optional when there is no replacement method.